### ISSUE TITLE
Fails only in travis because of unused standard

### ISSUE THREAD

**Issue by YerkoPalma** [depcheck/depcheck#360]
### Bug Description
Depcheck complaints about standard, but only happens in travis, manually running tests works locally.

**local tests**
![image](https://user-images.githubusercontent.com/5105812/60145808-3d8bbc80-9795-11e9-8da8-a34fbfee7d7c.png)

**travis tetst**
![image](https://user-images.githubusercontent.com/5105812/60183357-24febf00-97f4-11e9-957d-e57d9837c51c.png)

### Code snippets
I'm running my tests like this
```json
"test": "standard *.ts && depcheck --skip-missing=true --ignores=@types/*,@typescript-eslint/* && npm run test:integration",
```

### Versions
 - `node -v`: v12.4.0
 - `npm -v`: v6.9.0
 - `depcheck --version`: 0.8.1

### Extra info
The project is a typescript based electron app. Here is [the repo](https://github.com/YerkoPalma/todoapp). As you can see, in [the travis page](https://travis-ci.org/YerkoPalma/todoapp), standard is reported as unused, but appears in the test command above.

Here is the output of depcheck --json
<details>
<pre>
{
  "dependencies": [],
  "devDependencies": [
    "@types/dragula",
    "@types/tape",
    "@typescript-eslint/eslint-plugin"
  ],
  "missing": {
    "parse-cache-control": [
      "C:\\CLC\\VIDA\\Web\\Comun\\ypalma\\todoapp\\dist\\win-unpacked\\resources\\inspector\\audits2_worker\\audits2_worker_module.js"
    ],
    "semver": [
      "C:\\CLC\\VIDA\\Web\\Comun\\ypalma\\todoapp\\dist\\win-unpacked\\resources\\inspector\\audits2_worker\\audits2_worker_module.js"
    ],
    "lodash.isequal": [
      "C:\\CLC\\VIDA\\Web\\Comun\\ypalma\\todoapp\\dist\\win-unpacked\\resources\\inspector\\audits2_worker\\audits2_worker_module.js"
    ],
    "jpeg-js": [
      "C:\\CLC\\VIDA\\Web\\Comun\\ypalma\\todoapp\\dist\\win-unpacked\\resources\\inspector\\audits2_worker\\audits2_worker_module.js"
    ],
    "http-link-header": [
      "C:\\CLC\\VIDA\\Web\\Comun\\ypalma\\todoapp\\dist\\win-unpacked\\resources\\inspector\\audits2_worker\\audits2_worker_module.js"
    ],
    "axe-core": [
      "C:\\CLC\\VIDA\\Web\\Comun\\ypalma\\todoapp\\dist\\win-unpacked\\resources\\inspector\\audits2_worker\\audits2_worker_module.js"
    ],
    "robots-parser": [
      "C:\\CLC\\VIDA\\Web\\Comun\\ypalma\\todoapp\\dist\\win-unpacked\\resources\\inspector\\audits2_worker\\audits2_worker_module.js"
    ],
    "cssstyle": [
      "C:\\CLC\\VIDA\\Web\\Comun\\ypalma\\todoapp\\dist\\win-unpacked\\resources\\inspector\\audits2_worker\\audits2_worker_module.js"
    ],
    "metaviewport-parser": [
      "C:\\CLC\\VIDA\\Web\\Comun\\ypalma\\todoapp\\dist\\win-unpacked\\resources\\inspector\\audits2_worker\\audits2_worker_module.js"
    ],
    "lighthouse-logger": [
      "C:\\CLC\\VIDA\\Web\\Comun\\ypalma\\todoapp\\dist\\win-unpacked\\resources\\inspector\\audits2_worker\\audits2_worker_module.js"
    ],
    "speedline-core": [
      "C:\\CLC\\VIDA\\Web\\Comun\\ypalma\\todoapp\\dist\\win-unpacked\\resources\\inspector\\audits2_worker\\audits2_worker_module.js"
    ],
    "_process": [
      "C:\\CLC\\VIDA\\Web\\Comun\\ypalma\\todoapp\\dist\\win-unpacked\\resources\\inspector\\audits2_worker\\audits2_worker_module.js"
    ],
    "rimraf": [
      "C:\\CLC\\VIDA\\Web\\Comun\\ypalma\\todoapp\\dist\\win-unpacked\\resources\\inspector\\audits2_worker\\audits2_worker_module.js"
    ],
    "mkdirp": [
      "C:\\CLC\\VIDA\\Web\\Comun\\ypalma\\todoapp\\dist\\win-unpacked\\resources\\inspector\\audits2_worker\\audits2_worker_module.js"
    ],
    "intl-messageformat": [
      "C:\\CLC\\VIDA\\Web\\Comun\\ypalma\\todoapp\\dist\\win-unpacked\\resources\\inspector\\audits2_worker\\audits2_worker_module.js"
    ],
    "intl-messageformat-parser": [
      "C:\\CLC\\VIDA\\Web\\Comun\\ypalma\\todoapp\\dist\\win-unpacked\\resources\\inspector\\audits2_worker\\audits2_worker_module.js"
    ],
    "lookup-closest-locale": [
      "C:\\CLC\\VIDA\\Web\\Comun\\ypalma\\todoapp\\dist\\win-unpacked\\resources\\inspector\\audits2_worker\\audits2_worker_module.js"
    ],
    "intl": [
      "C:\\CLC\\VIDA\\Web\\Comun\\ypalma\\todoapp\\dist\\win-unpacked\\resources\\inspector\\audits2_worker\\audits2_worker_module.js"
    ],
    "raven": [
      "C:\\CLC\\VIDA\\Web\\Comun\\ypalma\\todoapp\\dist\\win-unpacked\\resources\\inspector\\audits2_worker\\audits2_worker_module.js"
    ],
    "inherits": [
      "C:\\CLC\\VIDA\\Web\\Comun\\ypalma\\todoapp\\dist\\win-unpacked\\resources\\inspector\\audits2_worker\\audits2_worker_module.js"
    ],
    "pako": [
      "C:\\CLC\\VIDA\\Web\\Comun\\ypalma\\todoapp\\dist\\win-unpacked\\resources\\inspector\\audits2_worker\\audits2_worker_module.js"
    ],
    "base64-js": [
      "C:\\CLC\\VIDA\\Web\\Comun\\ypalma\\todoapp\\dist\\win-unpacked\\resources\\inspector\\audits2_worker\\audits2_worker_module.js"
    ],
    "ieee754": [
      "C:\\CLC\\VIDA\\Web\\Comun\\ypalma\\todoapp\\dist\\win-unpacked\\resources\\inspector\\audits2_worker\\audits2_worker_module.js"
    ],
    "ms": [
      "C:\\CLC\\VIDA\\Web\\Comun\\ypalma\\todoapp\\dist\\win-unpacked\\resources\\inspector\\audits2_worker\\audits2_worker_module.js"
    ],
    "debug": [
      "C:\\CLC\\VIDA\\Web\\Comun\\ypalma\\todoapp\\dist\\win-unpacked\\resources\\inspector\\audits2_worker\\audits2_worker_module.js"
    ],
    "marky": [
      "C:\\CLC\\VIDA\\Web\\Comun\\ypalma\\todoapp\\dist\\win-unpacked\\resources\\inspector\\audits2_worker\\audits2_worker_module.js"
    ],
    "process-nextick-args": [
      "C:\\CLC\\VIDA\\Web\\Comun\\ypalma\\todoapp\\dist\\win-unpacked\\resources\\inspector\\audits2_worker\\audits2_worker_module.js"
    ],
    "core-util-is": [
      "C:\\CLC\\VIDA\\Web\\Comun\\ypalma\\todoapp\\dist\\win-unpacked\\resources\\inspector\\audits2_worker\\audits2_worker_module.js"
    ],
    "isarray": [
      "C:\\CLC\\VIDA\\Web\\Comun\\ypalma\\todoapp\\dist\\win-unpacked\\resources\\inspector\\audits2_worker\\audits2_worker_module.js"
    ],
    "safe-buffer": [
      "C:\\CLC\\VIDA\\Web\\Comun\\ypalma\\todoapp\\dist\\win-unpacked\\resources\\inspector\\audits2_worker\\audits2_worker_module.js"
    ],
    "util-deprecate": [
      "C:\\CLC\\VIDA\\Web\\Comun\\ypalma\\todoapp\\dist\\win-unpacked\\resources\\inspector\\audits2_worker\\audits2_worker_module.js"
    ],
    "image-ssim": [
      "C:\\CLC\\VIDA\\Web\\Comun\\ypalma\\todoapp\\dist\\win-unpacked\\resources\\inspector\\audits2_worker\\audits2_worker_module.js"
    ],
    "readable-stream": [
      "C:\\CLC\\VIDA\\Web\\Comun\\ypalma\\todoapp\\dist\\win-unpacked\\resources\\inspector\\audits2_worker\\audits2_worker_module.js"
    ],
    "acorn": [
      "C:\\CLC\\VIDA\\Web\\Comun\\ypalma\\todoapp\\dist\\win-unpacked\\resources\\inspector\\formatter_worker.js"
    ]
  },
  "using": {
    "parse-cache-control": [
      "C:\\CLC\\VIDA\\Web\\Comun\\ypalma\\todoapp\\dist\\win-unpacked\\resources\\inspector\\audits2_worker\\audits2_worker_module.js"
    ],
    "semver": [
      "C:\\CLC\\VIDA\\Web\\Comun\\ypalma\\todoapp\\dist\\win-unpacked\\resources\\inspector\\audits2_worker\\audits2_worker_module.js"
    ],
    "lodash.isequal": [
      "C:\\CLC\\VIDA\\Web\\Comun\\ypalma\\todoapp\\dist\\win-unpacked\\resources\\inspector\\audits2_worker\\audits2_worker_module.js"
    ],
    "jpeg-js": [
      "C:\\CLC\\VIDA\\Web\\Comun\\ypalma\\todoapp\\dist\\win-unpacked\\resources\\inspector\\audits2_worker\\audits2_worker_module.js"
    ],
    "http-link-header": [
      "C:\\CLC\\VIDA\\Web\\Comun\\ypalma\\todoapp\\dist\\win-unpacked\\resources\\inspector\\audits2_worker\\audits2_worker_module.js"
    ],
    "axe-core": [
      "C:\\CLC\\VIDA\\Web\\Comun\\ypalma\\todoapp\\dist\\win-unpacked\\resources\\inspector\\audits2_worker\\audits2_worker_module.js"
    ],
    "robots-parser": [
      "C:\\CLC\\VIDA\\Web\\Comun\\ypalma\\todoapp\\dist\\win-unpacked\\resources\\inspector\\audits2_worker\\audits2_worker_module.js"
    ],
    "cssstyle": [
      "C:\\CLC\\VIDA\\Web\\Comun\\ypalma\\todoapp\\dist\\win-unpacked\\resources\\inspector\\audits2_worker\\audits2_worker_module.js"
    ],
    "metaviewport-parser": [
      "C:\\CLC\\VIDA\\Web\\Comun\\ypalma\\todoapp\\dist\\win-unpacked\\resources\\inspector\\audits2_worker\\audits2_worker_module.js"
    ],
    "lighthouse-logger": [
      "C:\\CLC\\VIDA\\Web\\Comun\\ypalma\\todoapp\\dist\\win-unpacked\\resources\\inspector\\audits2_worker\\audits2_worker_module.js"
    ],
    "speedline-core": [
      "C:\\CLC\\VIDA\\Web\\Comun\\ypalma\\todoapp\\dist\\win-unpacked\\resources\\inspector\\audits2_worker\\audits2_worker_module.js"
    ],
    "_process": [
      "C:\\CLC\\VIDA\\Web\\Comun\\ypalma\\todoapp\\dist\\win-unpacked\\resources\\inspector\\audits2_worker\\audits2_worker_module.js"
    ],
    "rimraf": [
      "C:\\CLC\\VIDA\\Web\\Comun\\ypalma\\todoapp\\dist\\win-unpacked\\resources\\inspector\\audits2_worker\\audits2_worker_module.js"
    ],
    "mkdirp": [
      "C:\\CLC\\VIDA\\Web\\Comun\\ypalma\\todoapp\\dist\\win-unpacked\\resources\\inspector\\audits2_worker\\audits2_worker_module.js"
    ],
    "intl-messageformat": [
      "C:\\CLC\\VIDA\\Web\\Comun\\ypalma\\todoapp\\dist\\win-unpacked\\resources\\inspector\\audits2_worker\\audits2_worker_module.js"
    ],
    "intl-messageformat-parser": [
      "C:\\CLC\\VIDA\\Web\\Comun\\ypalma\\todoapp\\dist\\win-unpacked\\resources\\inspector\\audits2_worker\\audits2_worker_module.js"
    ],
    "lookup-closest-locale": [
      "C:\\CLC\\VIDA\\Web\\Comun\\ypalma\\todoapp\\dist\\win-unpacked\\resources\\inspector\\audits2_worker\\audits2_worker_module.js"
    ],
    "intl": [
      "C:\\CLC\\VIDA\\Web\\Comun\\ypalma\\todoapp\\dist\\win-unpacked\\resources\\inspector\\audits2_worker\\audits2_worker_module.js"
    ],
    "raven": [
      "C:\\CLC\\VIDA\\Web\\Comun\\ypalma\\todoapp\\dist\\win-unpacked\\resources\\inspector\\audits2_worker\\audits2_worker_module.js"
    ],
    "inherits": [
      "C:\\CLC\\VIDA\\Web\\Comun\\ypalma\\todoapp\\dist\\win-unpacked\\resources\\inspector\\audits2_worker\\audits2_worker_module.js"
    ],
    "pako": [
      "C:\\CLC\\VIDA\\Web\\Comun\\ypalma\\todoapp\\dist\\win-unpacked\\resources\\inspector\\audits2_worker\\audits2_worker_module.js"
    ],
    "base64-js": [
      "C:\\CLC\\VIDA\\Web\\Comun\\ypalma\\todoapp\\dist\\win-unpacked\\resources\\inspector\\audits2_worker\\audits2_worker_module.js"
    ],
    "ieee754": [
      "C:\\CLC\\VIDA\\Web\\Comun\\ypalma\\todoapp\\dist\\win-unpacked\\resources\\inspector\\audits2_worker\\audits2_worker_module.js"
    ],
    "ms": [
      "C:\\CLC\\VIDA\\Web\\Comun\\ypalma\\todoapp\\dist\\win-unpacked\\resources\\inspector\\audits2_worker\\audits2_worker_module.js"
    ],
    "debug": [
      "C:\\CLC\\VIDA\\Web\\Comun\\ypalma\\todoapp\\dist\\win-unpacked\\resources\\inspector\\audits2_worker\\audits2_worker_module.js"
    ],
    "marky": [
      "C:\\CLC\\VIDA\\Web\\Comun\\ypalma\\todoapp\\dist\\win-unpacked\\resources\\inspector\\audits2_worker\\audits2_worker_module.js"
    ],
    "process-nextick-args": [
      "C:\\CLC\\VIDA\\Web\\Comun\\ypalma\\todoapp\\dist\\win-unpacked\\resources\\inspector\\audits2_worker\\audits2_worker_module.js"
    ],
    "core-util-is": [
      "C:\\CLC\\VIDA\\Web\\Comun\\ypalma\\todoapp\\dist\\win-unpacked\\resources\\inspector\\audits2_worker\\audits2_worker_module.js"
    ],
    "isarray": [
      "C:\\CLC\\VIDA\\Web\\Comun\\ypalma\\todoapp\\dist\\win-unpacked\\resources\\inspector\\audits2_worker\\audits2_worker_module.js"
    ],
    "safe-buffer": [
      "C:\\CLC\\VIDA\\Web\\Comun\\ypalma\\todoapp\\dist\\win-unpacked\\resources\\inspector\\audits2_worker\\audits2_worker_module.js"
    ],
    "util-deprecate": [
      "C:\\CLC\\VIDA\\Web\\Comun\\ypalma\\todoapp\\dist\\win-unpacked\\resources\\inspector\\audits2_worker\\audits2_worker_module.js"
    ],
    "image-ssim": [
      "C:\\CLC\\VIDA\\Web\\Comun\\ypalma\\todoapp\\dist\\win-unpacked\\resources\\inspector\\audits2_worker\\audits2_worker_module.js"
    ],
    "readable-stream": [
      "C:\\CLC\\VIDA\\Web\\Comun\\ypalma\\todoapp\\dist\\win-unpacked\\resources\\inspector\\audits2_worker\\audits2_worker_module.js"
    ],
    "acorn": [
      "C:\\CLC\\VIDA\\Web\\Comun\\ypalma\\todoapp\\dist\\win-unpacked\\resources\\inspector\\formatter_worker.js"
    ],
    "electron-store": [
      "C:\\CLC\\VIDA\\Web\\Comun\\ypalma\\todoapp\\build\\main\\config.js",
      "C:\\CLC\\VIDA\\Web\\Comun\\ypalma\\todoapp\\build\\renderer\\lib\\todo-store.js",
      "C:\\CLC\\VIDA\\Web\\Comun\\ypalma\\todoapp\\src\\main\\config.ts",
      "C:\\CLC\\VIDA\\Web\\Comun\\ypalma\\todoapp\\src\\renderer\\lib\\todo-store.ts"
    ],
    "menubar": [
      "C:\\CLC\\VIDA\\Web\\Comun\\ypalma\\todoapp\\build\\main\\index.js",
      "C:\\CLC\\VIDA\\Web\\Comun\\ypalma\\todoapp\\src\\main\\index.ts"
    ],
    "electron-unhandled": [
      "C:\\CLC\\VIDA\\Web\\Comun\\ypalma\\todoapp\\build\\main\\index.js",
      "C:\\CLC\\VIDA\\Web\\Comun\\ypalma\\todoapp\\src\\main\\index.ts"
    ],
    "electron-debug": [
      "C:\\CLC\\VIDA\\Web\\Comun\\ypalma\\todoapp\\build\\main\\index.js",
      "C:\\CLC\\VIDA\\Web\\Comun\\ypalma\\todoapp\\src\\main\\index.ts"
    ],
    "electron-util": [
      "C:\\CLC\\VIDA\\Web\\Comun\\ypalma\\todoapp\\build\\main\\index.js",
      "C:\\CLC\\VIDA\\Web\\Comun\\ypalma\\todoapp\\build\\main\\menu.js",
      "C:\\CLC\\VIDA\\Web\\Comun\\ypalma\\todoapp\\src\\main\\index.ts",
      "C:\\CLC\\VIDA\\Web\\Comun\\ypalma\\todoapp\\src\\main\\menu.ts"
    ],
    "electron": [
      "C:\\CLC\\VIDA\\Web\\Comun\\ypalma\\todoapp\\build\\main\\index.js",
      "C:\\CLC\\VIDA\\Web\\Comun\\ypalma\\todoapp\\build\\main\\menu.js",
      "C:\\CLC\\VIDA\\Web\\Comun\\ypalma\\todoapp\\package.json",
      "C:\\CLC\\VIDA\\Web\\Comun\\ypalma\\todoapp\\src\\main\\index.ts",
      "C:\\CLC\\VIDA\\Web\\Comun\\ypalma\\todoapp\\src\\main\\menu.ts"
    ],
    "electron-reload": [
      "C:\\CLC\\VIDA\\Web\\Comun\\ypalma\\todoapp\\build\\main\\index.js",
      "C:\\CLC\\VIDA\\Web\\Comun\\ypalma\\todoapp\\src\\main\\index.ts"
    ],
    "nanohtml": [
      "C:\\CLC\\VIDA\\Web\\Comun\\ypalma\\todoapp\\build\\renderer\\lib\\prompt.js",
      "C:\\CLC\\VIDA\\Web\\Comun\\ypalma\\todoapp\\build\\renderer\\lib\\todo-item.js",
      "C:\\CLC\\VIDA\\Web\\Comun\\ypalma\\todoapp\\src\\renderer\\lib\\prompt.ts",
      "C:\\CLC\\VIDA\\Web\\Comun\\ypalma\\todoapp\\src\\renderer\\lib\\todo-item.ts"
    ],
    "dragula": [
      "C:\\CLC\\VIDA\\Web\\Comun\\ypalma\\todoapp\\build\\renderer\\lib\\todo-collection.js",
      "C:\\CLC\\VIDA\\Web\\Comun\\ypalma\\todoapp\\src\\renderer\\lib\\todo-collection.ts"
    ],
    "nanomorph": [
      "C:\\CLC\\VIDA\\Web\\Comun\\ypalma\\todoapp\\build\\renderer\\lib\\todo-item.js",
      "C:\\CLC\\VIDA\\Web\\Comun\\ypalma\\todoapp\\src\\renderer\\lib\\todo-item.ts"
    ],
    "depcheck": [
      "C:\\CLC\\VIDA\\Web\\Comun\\ypalma\\todoapp\\package.json"
    ],
    "electron-builder": [
      "C:\\CLC\\VIDA\\Web\\Comun\\ypalma\\todoapp\\package.json"
    ],
    "standard": [
      "C:\\CLC\\VIDA\\Web\\Comun\\ypalma\\todoapp\\package.json"
    ],
    "ts-node": [
      "C:\\CLC\\VIDA\\Web\\Comun\\ypalma\\todoapp\\package.json"
    ],
    "typescript": [
      "C:\\CLC\\VIDA\\Web\\Comun\\ypalma\\todoapp\\package.json"
    ],
    "@typescript-eslint/parser": [
      "C:\\CLC\\VIDA\\Web\\Comun\\ypalma\\todoapp\\package.json"
    ],
    "tape": [
      "C:\\CLC\\VIDA\\Web\\Comun\\ypalma\\todoapp\\test\\index.ts"
    ],
    "spectron": [
      "C:\\CLC\\VIDA\\Web\\Comun\\ypalma\\todoapp\\test\\setup.ts"
    ]
  },
  "invalidFiles": {},
  "invalidDirs": {}
}
</pre>
</details>

**Comment by rumpl** [comment]
Hi, thanks for reporting this!

Something is odd indeed. If a do a `npm ci` (like travis) I get the error, but there is no error if I do a `npm install` will look more.

Cheers

**Comment by rumpl** [comment]
not stale
